liberty-rest-protocol/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <!-- Parent -->
+  <parent>
+    <groupId>io.openliberty.arquillian</groupId>
+    <artifactId>arquillian-parent-liberty-jakarta</artifactId>
+    <version>2.0.3-SNAPSHOT</version>
```

Comment:
I know you are just matching the other versions here, but all the versions should be changed to 2.1-SNAPSHOT since we are adding new functionality I believe.

Reply:
Ok do we need to make a 2.0.x branch, or can I just do this where it is right now and change it to 2.1.0-SNAPSHOT without other changes?

Reply:
The main branch is the 2.x branch essentially. I was just pointing out that this does not feel like a minor fix release change where the third digit is bumped. So changing to 2.1.0-SNAPSHOT is preferred.

Reply:
Ok I have updated the POMs accordingly.  If they make a new alpha version of arquillian I would like to switch to that instead of having our own rest protocol implementation which is just a copy of what I put over there.  When that comes in this PR will get much smaller if I can use that instead.